### DISCUSSION TITLE
Podcasts sync - gpodder

### DISCUSSION THREAD

**Discussion by APraxx** [advplyr/audiobookshelf#1190] [Ideas]
Hi, 

i wanted to ask what you think of the possible integration of the [gpodder api](https://gpoddernet.readthedocs.io/en/latest/api/reference/) for podcast sync (subs, progress, etc)?
there are a few selfhosted implementations([mygpo](https://github.com/gpodder/mygpo),[nextcloud](https://github.com/thrillfall/nextcloud-gpodder)) and a cloud option with [gpodder.net](https://gpodder.net/). 

OT: thanks for the awesome app

**Comment by advplyr** [comment]
I'm not familiar with this. At first look it seems like an alternative to Abs then something we would integrate.
Can you be more specific on what you would like to see with this integration?

**Comment by APraxx** [comment]
I meant an implementation as a [client](https://gpoddernet.readthedocs.io/en/latest/user/clients.html#clients) who uses the api to sync state.
I see it kinda like whisper sync. You'd have the listening progress eg. on platforms you don't support or can use alternative clients without needing to update abs manually later. also most implementations seem to just use a subset eg. subscriptions and listened episodes   
   
maybe you are right and it overlaps too much therefore i started the discussion.

**Reply by APraxx** [reply]
> maybe you are right and it overlaps too much therefore i started the discussion.

maybe then it coud be seen as an superior export/import option over opml

**Comment by IcedQuinn** [comment]
It could be neat since Lissen refues to support SD cards, so smaller devices with those cards have to resort to other software. AntennaPod for example can pull books via the podcast API but you lose sync. (Antenna can sync via a local gpodder server, though this requires you do all the listening via podcast clients, so you can't switch between streaming on the desktop and caching chapters on a mobile when going outside.)

**Comment by BrayanLope5** [comment]
Gpodder support would be great as an option.

I really like the audiobookshelf android app but I don't like how complicated it makes it to manage podcasts. I have to always be connected to the server to still be able to listen to podcasts that I have not downloaded. 

AntennaPod is dedicated to podcasts and supports the gpodder api. It handles podcasts and playback really well. I love being able to also just have all my audiobooks, ebooks, and podcasts managed by audiobookshelf but then when out and about, right now, the app is less convenient for me.

Gpodder api would be great to have the option to use other apps. But I also like the app and can see the benefits to managing all three types of media from the singular app if it fit better with my use case.


P. S. Thank you for all the work you guys put into this project:)

**Comment by schtocker** [comment]
Would also love to see support for syncing with Gpodder. It would make ABS a significantly better podcast platform; currently the ABS Android app is not particularly good for listening to podcasts imo. With the Gpodder sync support, I could start a podcast on any app and pick it up, progress synced, on any other app or on the ABS web UI.

**Comment by mrkesu** [comment]
Any movement on this? I am also in the same boat where I would love to use different clients. For example the terminal client `podliner` also now supports this API and I'd love to be able to sync it with Audiobookshelf.

**Comment by pwinnski** [comment]
This seems like something that would be ideal as a plugin, rather than having ABS intrinsically add code for an unrelated service that in some ways overlaps with ABS itself. 

Once plugins are supported, of course. :)
